charter/index.html

```diff
@@ -291,92 +291,150 @@ <h1 id="title">
   </dl>
 </section>
 
-<section><h1>Community and Business Group Process</h1>
-  <p>
-    Terms of in this charter that conflict with those of the Community and
-    Business Group Process are void.
-  </p>
-  <section><h2>Work Limited to Charter Scope</h2>
-    <p>
-      The group will not publish Community Group Reports that are Specifications
-      on topics other than those listed under "Community Group Reports that are
-      Specifications" above. See below for how to modify the charter. The CLA
-      applies to these Community Group Reports.
-    </p>
+<section>
+    <h2>Community and Business Group Process</h2>
```

Comment:
FWIW, reformatting the HTML at the same time as you're changing the content, makes it really hard to see what's changed. Sometimes you can't avoid that, when you're reordering sections, but even then, it can help to reorder in one commit, and then reword in a separate one.